### DISCUSSION TITLE
error zarinpal

### DISCUSSION THREAD

**Discussion by rkmeans** [ali-zahedi/az-iranian-bank-gateways#123] [Q&A]
404 Client Error: Not Found for url: https://sandbox.zarinpal.com/pg/services/WebGate/wsdl

**Comment by birddevelper** [comment]
در لینک زیر که مربوط به تنظیمات لازم برای سندباکس زرین پال هست، لینک مورد نظر شما نیامده. و نیازی هم به این لینک ذکر نششده. 
میتونم بدونم برای چه کاری به این لینک نیاز دارید؟


آدرس هایی که نیاز هست به sandbox تغییر دهید

https://api.zarinpal.com/pg/v4/payment/request.json
https://api.zarinpal.com/pg/v4/payment/verify.json
https://www.zarinpal.com/pg/StartPay/

تغییر به سند باکس

https://sandbox.zarinpal.com/pg/v4/payment/request.json
https://sandbox.zarinpal.com/pg/v4/payment/verify.json
https://sandbox.zarinpal.com/pg/StartPay/

**Comment by rkmeans** [comment]
تشکر از پاسختان. من دارم جنگو کار میکنم و یک فروشگاه آنلاین نوشتم که قسمت
پرداختش را از گیت هاب شما سند باکس رزین استفاده کرده بودم و چند وقته کار
نمیکنه ولی قبلا کار میکرد لینکی که فرستادم را در متن برنامه zarinpal.py
خطهای آخرش نوشتید که خطا همانجاست و کار که نمی کند wsdl آدرس سند باکسش که
فرستادم کار نمیکند.
ممنون میشوم کمک کنید.
سپاس
